### PR TITLE
Remove the package Deprecated12, introduce package Deprecated13

### PR DESCRIPTION
**Pull request by pavel-krivanek**
Remove the package Deprecated12, introduce package Deprecated13

### CODE REVIEW

**Comment by jecisc** [comment]
This causes a lot of tests to fail. Some of the deprecated code is still used

**Comment by astares** [comment]
Would be good when the Deprecated13 package could be introduced nonetheless already.

**Comment by pavel-krivanek** [comment]
We need to fix these tests in NewTools
NewTools.Scopes.Tests.ScopesTest.testClassScopeExcludesClassCreatedAPosteriori	2.7 sec	1
NewTools.Scopes.Tests.ScopesTest.testPackageScopeIncludesClassCreatedAPosteriori

**Comment by pavel-krivanek** [comment]
https://github.com/pharo-spec/NewTools/pull/743

**Comment by pavel-krivanek** [comment]
The `RSDependencyTest>>#testDependencies` seems to fail in the vanilla image as well, not a problem of this PR

**Comment by pavel-krivanek** [comment]
and Roassal.Global.Tests.RSDependencyTest.testDependencies needs to be fixed in Roassal

**Comment by pavel-krivanek** [comment]
https://github.com/pharo-graphics/Roassal/pull/63

**Comment by pavel-krivanek** [comment]
God save integrators :-)

**Comment by astares** [comment]
One dependency is nuked with https://github.com/pharo-project/pharo/pull/16657

**Comment by jecisc** [comment]
Tests failures are down to 4

<a onclick="showFailureSummary('test-Unix64.ReleaseTests/ProperPackagesTest/unix_64___Tests_unix_64___testProperManifestCategorization/','Unix64.ReleaseTests/ProperPackagesTest/unix_64___Tests_unix_64___testProperManifestCategorization//summary')" id="test-Unix64.ReleaseTests/ProperPackagesTest/unix_64___Tests_unix_64___testProperManifestCategorization/-showlink" title="Show details"><svg class="icon-sm" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"></svg></a>
unix-64 / Tests-unix-64 / Unix64.ReleaseTests.ProperPackagesTest.testProperManifestCategorization | 0.34 sec | 1
-- | -- | --
 

 <a href="https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/testReport/Unix64.UnifiedFFI.Tests/FFIAutoReleaseOptionCalloutTest/unix_64___Tests_unix_64___testVoidPointer/">unix-64 / Tests-unix-64 / Unix64.UnifiedFFI.Tests.FFIAutoReleaseOptionCalloutTest.testVoidPointer</a>[unix-64 / Tests-unix-64 / Unix64.ReleaseTests.ProperPackagesTest.testProperManifestCategorization](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/testReport/Unix64.ReleaseTests/ProperPackagesTest/unix_64___Tests_unix_64___testProperManifestCategorization/)	0.34 sec	[1](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/)
 [unix-64 / Tests-unix-64 / Unix64.ReleaseTests.ReleaseTest.testThatThereAreNoSelectorsRemainingThatAreSentButNotImplemented](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/testReport/Unix64.ReleaseTests/ReleaseTest/unix_64___Tests_unix_64___testThatThereAreNoSelectorsRemainingThatAreSentButNotImplemented/)	1.2 sec	[1](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/)
 [unix-64 / Tests-unix-64 / Unix64.UnifiedFFI.Tests.FFIAutoReleaseOptionCalloutTest.testExternalStructure](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/testReport/Unix64.UnifiedFFI.Tests/FFIAutoReleaseOptionCalloutTest/unix_64___Tests_unix_64___testExternalStructure/)	8.6 sec	[1](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/)
 [unix-64 / Tests-unix-64 / Unix64.UnifiedFFI.Tests.FFIAutoReleaseOptionCalloutTest.testVoidPointer](https://ci.inria.fr/pharo-ci-jenkins2/job/Test%20pending%20pull%20request%20and%20branch%20Pipeline/job/PR-16538/4/testReport/Unix64.UnifiedFFI.Tests/FFIAutoReleaseOptionCalloutTest/unix_64___Tests_unix_64___testVoidPointer/)

**Comment by jecisc** [comment]
Fixing multiple calls to deprecated methods here: https://github.com/pharo-spec/NewTools/pull/749

**Comment by jecisc** [comment]
And here another one: https://github.com/pharo-project/pharo/pull/16658

File: src/Deprecated13/ManifestDeprecated13.class.st [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"
+Deprecated classes and methods of Pharo 13. 
+I will be removed in Pharo 14
+"
+Class {
+	#name : 'ManifestDeprecated13',
+	#superclass : 'PackageManifest',
+	#category : 'Deprecated13',
+	#package : 'Deprecated13'
```
Comment (jecisc) [inline src/Deprecated13/ManifestDeprecated13.class.st]:
```suggestion
	#package : 'Deprecated13',
	#tag: 'Manifest'
```

**Review by jecisc — CHANGES_REQUESTED** [review]
Requesting a change to fix one of the failing tests

**Comment by jecisc** [comment]
There is only one failure left that should be fixed with a PR I did to Pharo soon!

**Review by jecisc — APPROVED** [review]

**Pull request merged**